source/services/odnoklassniki.js

```diff
@@ -7,9 +7,7 @@ var config = require('../config'),
     dom    = require('../dom');
 
 var odnoklassniki = {
-    counterUrl: config.secure 
-        ? undefined 
-        : 'http://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
+    counterUrl: config.protocol + '://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
```

Comment:
@Corey-Maler maybe it should be just `//connect.ok.ru` without explicit protocol defining? In this case browsers use current protocol: if likely is used on https site that it will be https, otherwise will be http.